interface-spec.md

```diff
@@ -108,11 +108,17 @@ Triple is an alias of Quad.
 
 ### DataFactory
 
+For default values of the instance properties, see the individual [interface
+definitions](#data-interfaces)
+
 **Methods:**
 
 - `IRI .iri(String iri)` returns a new instance of IRI.
 - `BlankNode .blankNode()` returns a new instance of BlankNode.
-- `Literal .literal(String value, String language, String datatype)` returns a new instance of Literal.
+- Literal .literal(String value, [String languageOrDatatype]) returns a new
+  instance of Literal. If languageOrDatatype is an IRI, then a NamedNode is
```

Comment:
Relative IRIs cannot occur (and we should perhaps make this clear in the spec). It is the parser's job to always return absolute IRIs. If we'd allow relative IRIs, there would be no way to distinguish them from language strings, since all language strings are valid relative IRIs.

/:/ will match on any valid absolute IRI, including URNs (which need in fact two colons). It will not match on a language string. Hence, it is an acceptable test.